README.md

```diff
@@ -5,10 +5,28 @@ A template repository for creating data dashboards with Plotly.
 
 ## Getting started
 
-### How to use this template
+### Creating a copy of the template
 
 1. Choose a name for the repository to house your code. Please note this name should conform to [snake case (e.g. example_data_dashboard)](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841) to avoid annoying errors later!
 1. Create a new repository to house your dashboard from this template, instructions can be found [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template). **Do not overwrite this template repository.** 
+
+### Configuring GitHub policies
+By default, GitHub  does not apply any branch protection policies to newly created repositories. We use these policies to enforce things like: Requiring pull requests to commit changes into the main branch, requiring any comments on pull requests to be resolved and requiring status checks to pass before pull requests can be merged.
+
+1. Open the Settings menu for your GitHub repository
```

Comment:
Is it worth having a note to say that it has to be an admin of the repo? I guess they'd probably be doing it right after cloning so the person would always have access

Reply:
I tried cloning this myself into a test repo and I was assigned admin, but I'll add the comment just in case